http://sourceware.org/ml/binutils-cvs/2011-02/msg00156.html

### src/include/ChangeLog	2011/02/23 08:52:33	1.519
### src/include/ChangeLog	2011/02/28 18:30:14	1.520
## -1,3 +1,7 @@
+2011-02-28  Kai Tietz  <kai.tietz@onevision.com>
+
+	* filenames.h (filename_ncmp): New prototype.
+
 2011-02-23  Kai Tietz  <kai.tietz@onevision.com>
 
 	* dwarf2.h (_ELF_DWARF2_H): Renamed to
Index: gdb-7.2/include/filenames.h
===================================================================
--- gdb-7.2.orig/include/filenames.h
+++ gdb-7.2/include/filenames.h
@@ -73,6 +73,9 @@ extern "C" {
 extern int filename_cmp (const char *s1, const char *s2);
 #define FILENAME_CMP(s1, s2)	filename_cmp(s1, s2)
 
+extern int filename_ncmp (const char *s1, const char *s2,
+			  size_t n);
+
 #ifdef __cplusplus
 }
 #endif
